lib/rss/parser.rb

```diff
@@ -545,7 +545,7 @@ def setup_next_element(tag_name, klass, attributes)
         else
           if klass.have_content?
             if @last_element.need_base64_encode?
-              text = text.lstrip.unpack("m").first
+              text = text.lstrip.unpack1("m")
```

Comment:
```suggestion
              text = text.lstrip.unpack("m").first
```

Not in `Style/StringLiterals`.